diff --git a/node_modules/react-native-screens/ios/RNSScreen.mm b/node_modules/react-native-screens/ios/RNSScreen.mm
index b673407..bb3609e 100644
--- a/node_modules/react-native-screens/ios/RNSScreen.mm
+++ b/node_modules/react-native-screens/ios/RNSScreen.mm
@@ -614,22 +614,24 @@ - (void)updatePresentationStyle
 #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_15_0) && \
     __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_15_0
   if (@available(iOS 15.0, *)) {
-    UISheetPresentationController *sheet = _controller.sheetPresentationController;
-    if (_stackPresentation == RNSScreenStackPresentationFormSheet && sheet != nil) {
-      sheet.prefersScrollingExpandsWhenScrolledToEdge = _sheetExpandsWhenScrolledToEdge;
-      sheet.prefersGrabberVisible = _sheetGrabberVisible;
-      sheet.preferredCornerRadius =
-          _sheetCornerRadius < 0 ? UISheetPresentationControllerAutomaticDimension : _sheetCornerRadius;
-
-      if (_sheetLargestUndimmedDetent == RNSScreenDetentTypeMedium) {
-        sheet.largestUndimmedDetentIdentifier = UISheetPresentationControllerDetentIdentifierMedium;
-      } else if (_sheetLargestUndimmedDetent == RNSScreenDetentTypeLarge) {
-        sheet.largestUndimmedDetentIdentifier = UISheetPresentationControllerDetentIdentifierLarge;
-      } else if (_sheetLargestUndimmedDetent == RNSScreenDetentTypeAll) {
-        sheet.largestUndimmedDetentIdentifier = nil;
-      } else {
-        RCTLogError(@"Unhandled value of sheetLargestUndimmedDetent passed");
-      }
+      if (_stackPresentation == RNSScreenStackPresentationFormSheet) {
+          UISheetPresentationController *sheet = _controller.sheetPresentationController;
+          if (sheet != nil) {
+              sheet.prefersScrollingExpandsWhenScrolledToEdge = _sheetExpandsWhenScrolledToEdge;
+              sheet.prefersGrabberVisible = _sheetGrabberVisible;
+              sheet.preferredCornerRadius =
+              _sheetCornerRadius < 0 ? UISheetPresentationControllerAutomaticDimension : _sheetCornerRadius;
+
+              if (_sheetLargestUndimmedDetent == RNSScreenDetentTypeMedium) {
+                  sheet.largestUndimmedDetentIdentifier = UISheetPresentationControllerDetentIdentifierMedium;
+              } else if (_sheetLargestUndimmedDetent == RNSScreenDetentTypeLarge) {
+                  sheet.largestUndimmedDetentIdentifier = UISheetPresentationControllerDetentIdentifierLarge;
+              } else if (_sheetLargestUndimmedDetent == RNSScreenDetentTypeAll) {
+                  sheet.largestUndimmedDetentIdentifier = nil;
+              } else {
+                  RCTLogError(@"Unhandled value of sheetLargestUndimmedDetent passed");
+              }
+          }
 
       if (_sheetAllowedDetents == RNSScreenDetentTypeMedium) {
         sheet.detents = @[ UISheetPresentationControllerDetent.mediumDetent ];
