diff --git a/node_modules/react-native-reanimated/RNReanimated.podspec b/node_modules/react-native-reanimated/RNReanimated.podspec
index 67d3a7f..472eee1 100644
--- a/node_modules/react-native-reanimated/RNReanimated.podspec
+++ b/node_modules/react-native-reanimated/RNReanimated.podspec
@@ -103,6 +103,6 @@ Pod::Spec.new do |s|
   end
 
   if config[:react_native_minor_version] >= 72 && !fabric_enabled
-    s.dependency 'React-RCTAppDelegate'
+    s.dependency 'React-jsc'
   end
 end
